paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -571,15 +571,15 @@ class Paparazzi @JvmOverloads constructor(
     private val savedStateRegistryController = SavedStateRegistryController.create(this)
 
     override fun getLifecycle(): Lifecycle = lifecycleRegistry
-    override fun getSavedStateRegistry(): SavedStateRegistry = savedStateRegistryController.savedStateRegistry
+    override val savedStateRegistry: SavedStateRegistry = savedStateRegistryController.savedStateRegistry
```

Comment:
Does this mean anyone using version this PR goes into, will be required to use Compose 1.2.0? (similar to "support" vs "require"). Also based on this it looks like anyone using Compose 1.1.1 cannot use Paparazzi? (and probably that was your motivation with the PR)

Reply:
Yes!
My bad, I'll update this info in PR description.
Currently in our project we live with sources builded from this MR in our local maven.

Reply:
Description and title for PR updated.
Thanks for comment!